distributed/batched.py

```diff
@@ -58,6 +58,7 @@ def __init__(self, interval, loop=None, serializers=None):
 
     def start(self, comm):
         self.comm = comm
+        self.please_stop = False
         self.loop.add_callback(self._background_send)
```

Comment:
One downside of using Tornado here is that we have no handle on this coroutine. If we did `asyncio.create_task` or similar, we could keep a Task object here.
1. That might let us simplify the `please_stop`/`stopped`/`abort` logic by just letting us do `task.cancel()` and handling the `CancelledError` within `_background_send`. We would probably want to `shield` the `comm.write` call from cancellation if we did so.
2. We could assert in `send` and elsewhere that the `_background_send` Task is not `done()`. If we're also checking `please_stop`, etc. this may be overkill, but it's a nice sanity check that something hasn't gone horribly wrong. The fact that the coroutine wasn't running was the cause of https://github.com/dask/distributed/issues/5481; a check for this would have made the bug much easier to find.